### PR TITLE
[Workers] Document Workers route limit

### PR DESCRIPTION
**Pull request by WalshyDev**
Documenting WC-929

### CODE REVIEW

File: content/workers/platform/limits.md [hunk 1]
```diff
@@ -276,6 +276,10 @@ App Workers do not count towards this limit.
 
 {{</Aside>}}
 
+## Number of routes
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
## Number of routes per zone
```

File: content/workers/platform/limits.md [hunk 2]
```diff
@@ -276,6 +276,10 @@ App Workers do not count towards this limit.
...
+
+Each zone has a limit of 1,000 [routes](https://developers.cloudflare.com/workers/platform/triggers/routes/). If you require more than this you may want to consider [Workers for Platforms](https://developers.cloudflare.com/cloudflare-for-platforms/workers-for-platforms/) or reducing the number of routes required. You may also request an increase by completing the [Limit Increase Request Form](https://forms.gle/ukpeZVLWLnKeixDu7).
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
Each zone has a limit of 1,000 [routes](https://developers.cloudflare.com/workers/platform/triggers/routes/). If you require more than 1,000 routes on your zone, consider using [Workers for Platforms](https://developers.cloudflare.com/cloudflare-for-platforms/workers-for-platforms/) or request an increase to this limit by completing the [Limit Increase Request Form](https://forms.gle/ukpeZVLWLnKeixDu7).
```

Comment (WalshyDev) [inline content/workers/platform/limits.md]:
```suggestion
Each zone has a limit of 1,000 [routes](/workers/platform/triggers/routes/). If you require more than this you may want to consider [Workers for Platforms](/cloudflare-for-platforms/workers-for-platforms/) or reducing the number of routes required. You may also request an increase by completing the [Limit Increase Request Form](https://forms.gle/ukpeZVLWLnKeixDu7).
```

**Review by deadlypants1973 — APPROVED** [review]
thanks walshy < 3

File: content/workers/platform/limits.md [hunk 3]
```diff
@@ -276,6 +276,10 @@ App Workers do not count towards this limit.
 
 {{</Aside>}}
 
+## Number of routes per zone
+
+Each zone has a limit of 1,000 [routes](https://developers.cloudflare.com/workers/platform/triggers/routes/). If you require more than 1,000 routes on your zone, consider using [Workers for Platforms](https://developers.cloudflare.com/cloudflare-for-platforms/workers-for-platforms/) or request an increase to this limit by completing the [Limit Increase Request Form](https://forms.gle/ukpeZVLWLnKeixDu7).
```
Comment (WalshyDev) [inline content/workers/platform/limits.md]:
```suggestion
Each zone has a limit of 1,000 [routes](/workers/platform/triggers/routes/). If you require more than 1,000 routes on your zone, consider using [Workers for Platforms](/cloudflare-for-platforms/workers-for-platforms/) or request an increase to this limit by completing the [Limit Increase Request Form](https://forms.gle/ukpeZVLWLnKeixDu7).
```

**Pull request merged**